plugins/woocommerce/client/admin/client/marketing/data-multichannel/selectors.ts

```diff
@@ -1,3 +1,8 @@
+/**
+ * External dependencies
+ */
+import { WPDataSelector, WPDataSelectors } from '@woocommerce/data';
```

Comment:
We can explicitly import the type:

```suggestion
import type { WPDataSelector, WPDataSelectors } from '@woocommerce/data';
```